requirements_core_min.txt

```diff
@@ -1,3 +1,3 @@
-# https://github.com/home-assistant/core/blob/2023.6.0/homeassistant/components/frontend/manifest.json
-home-assistant-frontend==20230607.0
-homeassistant==2023.6.0
+# https://github.com/home-assistant/core/blob/2024.4.0/homeassistant/components/frontend/manifest.json
+home-assistant-frontend==20240403.1
+homeassistant==2024.4.0
```

Comment:
```suggestion
homeassistant==2024.4.1
```
This also needs to be adjusted in `hacs.json`

Reply:
Fixed in https://github.com/hacs/integration/pull/3672/commits/92d117a35371d69074c51d395e39de831606fdcd